pkg/lightning/backend/local/local.go

```diff
@@ -434,7 +820,7 @@ func NewLocalBackend(
 	g glue.Glue,
 	maxOpenFiles int,
 ) (backend.Backend, error) {
-	localFile := cfg.SortedKVDir
+	File := cfg.SortedKVDir
```

Comment:
why rename the variable. as least use a lower case name.

Reply:
This must be done by the IDE when I refactor the `LocalFile` struct..

Reply:
![1-fs8](https://user-images.githubusercontent.com/103023/116520391-3e4b8780-a905-11eb-92a1-a71f3d87cc0a.png)

Reply:
😂😂😂